src/features/groups/components/ContainerList.tsx

```diff
@@ -9,7 +12,14 @@ export default async function ContainerList({ id }: { id: string }) {
       {/* <MenuMeatBall /> */}
       {/* <MenuPlus /> */}
       {containers.map((container) => (
-        <div key={container.id}>{container.name},</div>
+        <Card key={container.id}>
+          <div className="flex gap-4 items-center">
+            <span className="flex items-center justify-center bg-accent rounded-full w-11 h-11">
+              <Icon iconName="container" color="black" />
+            </span>
+            <span>{container.name}</span>
```

Comment:
After conducting some research, I found that both `<div>` and `<span>` tags are semantically neutral. However, when used as a child element of a flexbox, `<div>` tags are commonly used for creating layouts, while `<span>` tags are usually used to wrap text or other inline elements.

Based on this information, my suggestion is to use `<div>` to wrap the `<Icon>` tag, and use `<span>` (or no tags at all) for `{container.name}`. If you do not apply any styles or attributes to the `<span>` tag, you do not even need to enclose the content with it.

Reply:
Thank you for checking!! Fixed!!